### DISCUSSION TITLE
In a deep object is there a way to access the parent

### DISCUSSION THREAD

**Discussion by caleb-cushing0-kr** [mapstruct/mapstruct#3505] [Q&A]
given `foo().bar().baz()` where `Baz` does not contain a reference to `Bar parent`, is it possible for mapstruct to pass in a copy of `Bar parent` to the mapping method?

**Comment by filiphr** [comment]
@caleb-cushing0-kr I am not sure that I understand your question. Where would you like MapStruct to pass a copy of the `parent`? Can you perhaps provide a more meaningful example with more code?
